test/service/test_manifest.py

```diff
@@ -1678,12 +1681,11 @@ def test_fetch_manifest(self, get_cached_manifest):
         for format_ in ManifestFormat:
             with self.subTest(format_=format_):
                 object_key = 'some_object_key'
-                url = furl(url=self.base_url,
-                           path='/manifest/files',
-                           args=dict(catalog=self.catalog,
-                                     format=format_.value,
-                                     filters='{}',
-                                     objectKey=object_key)).url
+                url = self.base_url.set(path='/manifest/files',
+                                        args=dict(catalog=self.catalog,
+                                                  format=format_.value,
+                                                  filters='{}',
```

Comment:
>In the previous section, we used an actual dictionary for the filters arg, but here we use a string literal. Is there a reason for doing so? 

In this case yes. It appears that it gets removed from the args if it is an empty dict, since we use this URL for asserting the url in the response, the `'{}'` must be present due to its presence in the response's URL. That being said, I removed it from the other sections since it isn't needed.